tests/containers/podman_image.pm

```diff
@@ -19,24 +19,25 @@ use containers::common;
 use containers::container_images;
 use suse_container_urls 'get_suse_container_urls';
 use version_utils qw(get_os_release check_os_release);
+use containers::utils 'can_build_sle_base';
 
 sub run {
     my ($image_names, $stable_names) = get_suse_container_urls();
     my ($running_version, $sp, $host_distri) = get_os_release;
-    my $runtime = "podman";
+    my $container_engine = "podman";
     install_podman_when_needed($host_distri);
-    allow_selected_insecure_registries(runtime => $runtime);
+    allow_selected_insecure_registries(runtime => $container_engine);
     for my $iname (@{$image_names}) {
-        test_container_image(image => $iname, runtime => $runtime);
-        build_container_image(image => $iname, runtime => $runtime);
+        test_container_image(image => $iname, runtime => $container_engine);
+        build_container_image(image => $iname, runtime => $container_engine) if can_build_sle_base;
```

Comment:
Container images can be build on non-SLE distribution too.

Reply:
yes but not sle base